.github/workflows/pr_iac_lint.yml

```diff
@@ -0,0 +1,45 @@
+name: Validate IaC
+
+on:
```

Comment:
We should add:
```
push:
  branches:
    - main
    - v2 
```

so that we run this when we have PRs merged down into the two tip branches too